internal/ui/table.go

```diff
@@ -288,6 +288,26 @@ func (t *Table) SetSortCol(name string, asc bool) {
 	t.setSortCol(model1.SortColumn{Name: name, ASC: asc})
 }
 
+func (t *Table) GetSortCol() (string, bool) {
+	return t.sortCol.Name, t.sortCol.ASC
+}
+
+func (t *Table) isVisible(h model1.HeaderColumn) bool {
```

Comment:
Nice refactor! I think this should also check for hide/wide attribute on the column.
Also we should reuse this call on buildRow below.